engine/Assets/Scripts/Modes/PracticeMode.cs

```diff
@@ -139,16 +139,17 @@ public class PracticeMode : IMode {
             _showingScoreboard = true;
             DynamicUIManager.CreatePanel<ScoreboardPanel>(true, false);
         }
-        bool openEscapeMenu = InputManager.MappedValueInputs[TOGGLE_ESCAPE_MENU_INPUT].Value == 1.0F;
+        // TODO: Fix this error
```

Comment:
Is this important?

Reply:
Oh yeah, I was getting the escape button error again. I will fix that